app/pages/BeelieversAuction/MintInfo.tsx

```diff
@@ -73,19 +96,30 @@ function MintAction({ refund }: MintActionProps) {
 	};
 
 	return (
-		<div className="flex gap-4">
-			<Button
-				type="button"
-				disabled={isPending}
-				onClick={() => {
-					// TODO: handle mint
-				}}
-			>
-				Mint
-			</Button>
-			{refund && (
-				<Button type="button" disabled={isPending} onClick={handleRefund}>
-					Refund {formatSUI(refund)} SUI
+		<div className="flex flex-col sm:flex-row gap-4">
+			{isUserEligibleForMinting && (
+				<Button
+					type="button"
+					disabled={isPending}
+					size="lg"
+					className="flex-1 bg-primary hover:bg-primary/90 text-white font-semibold py-3 px-6 rounded-lg shadow-lg hover:shadow-xl transition-all duration-300"
+					onClick={() => {
+						// TODO: handle mint
+					}}
+				>
+					🐝 Mint
+				</Button>
+			)}
+			{isUserEligibleForRefund && refund && (
+				<Button
+					type="button"
+					disabled={isPending}
+					size="lg"
+					variant="outline"
+					className="flex-1 border-primary text-primary hover:bg-primary hover:text-white font-semibold py-3 px-6 rounded-lg shadow-lg hover:shadow-xl transition-all duration-300"
```

Comment:
I think we already have a standard class for buttons. Please reuse it , rather than defining all those classes

Reply:
Done.